<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleParagraphWidget</class>
 <widget class="QWidget" name="SimpleParagraphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>66</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="0">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="2" column="0" colspan="2">
      <widget class="FormattingButton" name="bulletListButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="FormattingButton" name="numberedListButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QToolButton" name="decreaseIndent">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="6" colspan="2">
      <widget class="QToolButton" name="increaseIndent">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QToolButton" name="changeTextDirection"/>
     </item>
     <item row="2" column="9">
      <widget class="QuickTableButton" name="quickTable">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QToolButton" name="alignLeft">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QToolButton" name="alignCenter">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QToolButton" name="alignRight">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="5" colspan="2">
      <widget class="QToolButton" name="alignBlock">
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="7" colspan="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="11">
      <widget class="StylesCombo" name="paragraphStyleCombo">
      </widget>
     </item>
     <item row="2" column="10">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="SpecialSpacer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QuickTableButton</class>
   <extends>QToolButton</extends>
   <header>dialogs/QuickTableButton.h</header>
  </customwidget>
  <customwidget>
   <class>FormattingButton</class>
   <extends>QToolButton</extends>
   <header>dialogs/FormattingButton.h</header>
  </customwidget>
  <customwidget>
   <class>StylesCombo</class>
   <extends>QComboBox</extends>
   <header>dialogs/StylesCombo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
